cmd/copyUtil.go

```diff
@@ -324,6 +324,12 @@ func (util copyHandlerUtil) isPathALocalDirectory(pathString string) bool {
 
 func (util copyHandlerUtil) generateLocalPath(directoryPath, fileName string) string {
 	var result string
+
+	//Because \ and / can't be used interchangeably with the long file path discriminator (\\?\), filter forward slash out
+	if strings.HasPrefix(directoryPath, `\\?\`) {
```

Comment:
>\\?\` [](start = 38, length = 5)

Can we have a constant defined for this, and the use that constant everywhere instead of the literal string?  Then, we can put a comment on the constant explaining it to folks who aren't familiar with Window's approach to long paths.